doc/setup/email_gateway.rst

```diff
@@ -0,0 +1,40 @@
+:banner: banners/email_gateway.jpg
+
+==================
+Odoo email gateway
+==================
+
+The Odoo mail gateway is a better alternative than using an "incoming mail server"
+because the messages are received instantaneously.
```

Comment:
The Odoo mail gateway allows you to inject directly all the received emails in Odoo.